java/ql/src/semmle/code/java/frameworks/Castor.qll

```diff
@@ -0,0 +1,20 @@
+/**
+ * Provides classes and predicates for working with the Castor framework.
+ */
+
+import java
+
+/**
+ * The class `org.exolab.castor.xml.Unmarshaller`.
+ */
+class CastorUnmarshaller extends RefType {
+  CastorUnmarshaller() { this.hasQualifiedName("org.exolab.castor.xml", "Unmarshaller") }
+}
+
+/** A method with the name `unmarshal` declared in `org.exolab.castor.xml.Unmarshaller`. */
+class UnmarshalMethod extends Method {
```

Comment:
This name is a little too generic, let's include `Castor` in the name:
```suggestion
class CastorUnmarshalMethod extends Method {
```

Reply:
I made a change.